### DISCUSSION TITLE
Cosmological parameters

### DISCUSSION THREAD

**Discussion by subhratadey** [mboquien/cigale#110] [Q&A]
What are the cosmological constant and parameters used in CIGALE? Also is it possible to change or update them? I want to use the values given by Planck 2018 data release (https://ui.adsabs.harvard.edu/abs/2020A%26A...641A...6P/abstract)

**Comment by mboquien** [comment]
Hello. The upcoming release of CIGALE will use the Planck 2018 cosmology. If you want to do so with the current version, you can just apply [this change](https://gitlab.lam.fr/cigale/cigale/-/commit/58f2c4b19ff1615e50f8f4149fa728ef80a80e4a#6a37c81b5e8267b897c080fa9ad3f3f52b7d3a69) to `pcigale/utils/cosmology.py`. Then you should be good to go.

**Reply by subhratadey** [reply]
Thanks!
